community/docker/APKBUILD

```diff
@@ -27,12 +27,14 @@ _runc_ver=a01dafd48bc1c7cc12bdb01206f9fea7dd6feb70
 _containerd_ver=977c511eda0925a723debdc94d09459af49d082a
 _libnetwork_ver=0f534354b813003a754606689722fe253101bc4e
 _tini_ver=949e6facb77383876aeff8a6944dde66b3089574
+_tini_gitver=949e6fa
```

Comment:
@ncopa I need to look and see how to extract the first 7 characters from the version string.